omod/src/main/resources/liquibase.xml

```diff
@@ -195,5 +195,31 @@
 							 tableName="metadatamapping_metadata_source"
 							 columnNames="name" />
 	</changeSet>
-
+	
+	<changeSet id="metadatamapping-2016-01-06-0800" author="jasonvena">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="metadatamapping_metadata_term_mapping"
+						  columnName="metadata_class" />
+		</preConditions>
```

Comment:
What this precondition means is: if the column metadata_class does not exist, mark this changeset as having been already ran and don't do anything else. Perhaps you meant to check if the constraint exists? Btw, I'm personally not sure if this kind of preconditions are useful or not but they seem to be a standard practice in OpenMRS and I guess it's a good idea to follow the established practices.

http://www.liquibase.org/documentation/preconditions.html